bucket/ryzen-controller.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.5.4",
+    "description": "Utility for tweaking AMD Ryzen laptops.",
+    "homepage": "https://ryzencontroller.com/",
+    "license": {
+        "identifier": "CC0-1.0",
+        "url": "https://gitlab.com/ryzen-controller-team/ryzen-controller/-/blob/master/LICENSE.md"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://gitlab.com/ryzen-controller-team/ryzen-controller/-/jobs/1710820449/artifacts/raw/dist/win/Ryzen%20Controller%20Setup%202.5.4.exe#/dl.7z",
+            "hash": "28babc45e0a629d18f75d2e32e8dd541289fdd0df51686cd8f3032e038f89616"
+        }
+    },
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall Ryzen Controller.exe\" -Force -Recurse | Out-Null"
+    ],
+    "shortcuts": [
+        [
+            "Ryzen Controller.exe",
+            "Ryzen Controller"
+        ]
+    ],
+    "checkver": "(?sm)Windows 10.*?Version ([\\d.]+) via GitLab"
```

Comment:
You can use this URL

https://gitlab.com/ryzen-controller-team/ryzen-controller/-/tags?sort=name_desc&format=atom

Maybe use XMLPath or something?

Reply:
Thanks for the info.

I tried using XPath, but it seems that the XML cannot be handled properly by *Scoop*. I think it is related to this line: `<feed xmlns="http://www.w3.org/2005/Atom" xmlns:media="http://search.yahoo.com/mrss/">`

I am now using *regex* instead.
Will investigate about the bug later.

log:
```
MethodInvocationException: D:\scoop\apps\scoop\current\bin\checkver.ps1:234
Line |
 234 |                  $nsmgr.AddNamespace('xmlns', $_.Value)
     |                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "AddNamespace" with "2" argument(s): "Prefix "xmlns" is reserved for use by XML."

ryzen-controller: couldn't find '/feed/entry[1]/title' in https://gitlab.com/ryzen-controller-team/ryzen-controller/-/tags?sort=name_desc&format=atom
```